app/controllers/report_controller/reports/editor.rb

```diff
@@ -525,7 +525,9 @@ def gfv_model
         @edit[:new][:tz] = session[:user_tz]
         build_perf_interval_arrays(@edit[:new][:perf_interval]) # Build the start and end arrays for the performance interval chooser
       end
-      if model_report_type(@edit[:new][:model]) == :chargeback
+      if model_report_type(@edit[:new][:model]).to_s.starts_with? "chargeback"
+        @edit[:new][:cb_model] = "Vm" if @edit[:new][:model] == "ChargebackVm"
```

Comment:
how about changing this to @edit[:new][:cb_model] = @edit[:new][:model] == "ChargebackVm" ? "Vm" :  "ContainerProject"